### ISSUE TITLE
Table Block: Needs more tools/options

### ISSUE THREAD

**Issue by bhubbard** [WordPress/gutenberg#57278]
## What problem does this address?
Currently I am unable to do the following in the table block:

- Style the Table headers separate from rest of the table, ie. different colors/fonts etc
- Style the Table footers separate from rest of the table, ie. different colors/fonts etc
- Style the Table caption separate from rest of the table, ie. different colors/fonts etc
- No way to merge cells with a `colspan` or `rowspan` without editing html
- No way to style individual cells separately from the column, ie. text align center for one cell, and left align for the rest. border, margin, etc for individual cells

## What is your proposed solution?
Add support for these, maybe with some tool that is a cell selector under block style settings? Maybe we need child blocks such as `table header block`, `table footer block`, etc

**Comment by talldan** [comment]
It does need some love, but I think there are issues for all these things already.

**Comment by colorful-tones** [comment]
Yes, I'm quite certain most of the requested features have already been noted in prior open issues: https://github.com/WordPress/gutenberg/labels/%5BBlock%5D%20Table

Some relevant issues that overlap here:

- https://github.com/WordPress/gutenberg/issues/6923
- https://github.com/WordPress/gutenberg/issues/15821
- https://github.com/WordPress/gutenberg/issues/16355

I suggest we close this issue as it is a duplicate. Thoughts? 🤔

**Comment by youknowriad** [comment]
Closing as a duplicate. These requests are tracked in [#6923](https://github.com/WordPress/gutenberg/issues/6923) (table editor features) and #15821 (merge/unmerge cells). Please add any missing detail there.
